dev/tools/controllerbuilder/README.md

```diff
@@ -0,0 +1,17 @@
+Examples:
```

Comment:
This feels a bit off. It seems like its copied from the generate.sh file. Do we need to keep this up to date? What do I learn here that I wouldn't get from the script and if there is something maybe we should highlight it? If we want to explain to someone how to run a generate an individual resource, maybe we should package the script into individual resources to make that easy and then have a top level which runs them all?

Reply:
Agreed - they should be in generate.sh, and we will DRY generate.sh as we add more types.

I feel like I keep deleting this file, it just won't go away!  Deleted!